systemd/atuin.service

```diff
@@ -0,0 +1,29 @@
+[Unit]
```

Comment:
Could we also use the name `atuin-server` please? I think it's fairly important to make the distinction, especially considering the Atuin Daemon that will soon also have a systemd unit file

Reply:
Alright, renamed this and the sysusers file.